### PR TITLE
add data binding product view inventory

### PR DESCRIPTION
**Pull request by stephaniepark**

### CODE REVIEW

**Comment by JacobRintjema** [comment]
lgtm !1!!11!

**Comment by johnswyoon** [comment]
taking a look as well gimme a sec

File: FarmBuddy/app/src/main/java/com/example/farmbuddy/ui/productview/ProductViewFragment.kt [hunk 1]
```diff
@@ -50,10 +52,32 @@ class ProductViewFragment : Fragment() {
...
+        val adapter = ProductProduceAdapter(Product())
+        binding.inventorySection.adapter = adapter
+
+        // Getting Product Data from Firestore
+        val userDBHandler = UserDBHandler()
+        userDBHandler.getInventory { inventory : MutableList<Product>? ->
+            requireActivity().runOnUiThread {
+                if (inventory != null) {
+                    //TODO: link with correct data
+                    adapter.setData(inventory, "1ab1f3a6-231c-46c9-a9c5-48e73c0aa6f0")
```
Comment (JacobRintjema) [inline FarmBuddy/app/src/main/java/com/example/farmbuddy/ui/productview/ProductViewFragment.kt:67]: what is this uuid string?

Comment (JacobRintjema) [reply FarmBuddy/app/src/main/java/com/example/farmbuddy/ui/productview/ProductViewFragment.kt:67]: this just for testing purposes i'm guessing?

**Review by johnswyoon — APPROVED** [review]
lgtm

**Pull request merged**